### DISCUSSION TITLE
wifi_command_preview_stream.py Error: 409 Client Error: Conflict for url

### DISCUSSION THREAD

**Discussion by leongthb** [gopro/OpenGoPro#280] [General]
![image](https://user-images.githubusercontent.com/116328546/210027397-681968e3-2938-4437-94df-5608bacc48ac.png)

OS: Ubuntu 22.04.1 LTS on VMware
Python 3.10.6
bleak: Version: 0.15.1
bluez: Version: 5.64-0ubuntu1
protobuf: Version: 4.21.12
libprotoc 3.12.4
requests: 2.28.1

Script: OpenGoPro-main\demos\python\tutorial\tutorial_modules\tutorial_6_send_wifi_commands\wifi_command_preview_stream.py

When I run wifi_command_preview_stream.py, it works, 
but when I run the script next day, 
http://10.5.5.9:8080/gopro/camera/stream/stop works,
http://10.5.5.9:8080/gopro/camera/stream/start shows error: 409 Client Error: Conflict for url

I have tried journalctl, btmon, there is no infomation about this error. 

Do anyone encounter this error before? I would appreaciate it very much if anyone could give any advice on how to get more infomation about this error. thanks.

**Comment by tcamise-gpsw** [comment]
Hello. I can reproduce the 409 status by performing a Preview Start when the Preview is already enabled.

So perhaps you did not successfully stop the Preview?

Note that you can check the current status of the Preview Stream with the [Status ID 32](https://gopro.github.io/OpenGoPro/http_2_0#status-ids)

**Reply by leongthb** [reply]
Thanks for your reply, the script (wifi_command_preview_stream.py) first perform "stop Preview", the result shows stopped successfully, then start the preview, and report "error 409 conflict url", besides that error message, I don't know any method to get more information about this error. I reset the connection of gopro, the error has disappeared, but it will reappear sometimes later, I will check the status when the 409 error reappears. thank you!
